src/lib.rs

```diff
@@ -783,7 +783,6 @@ impl Config {
                 let mut config = pkg_config::Config::new();
                 config
                     .print_system_libs(false)
-                    .cargo_metadata(false)
```

Comment:
Do we still need this change? As said, I'd rather not remove the metadata as it will likely break things.

Reply:
IIRC no, this can be reverted safely.